packages/office-ui-fabric-react/src/components/DetailsList/DetailsRowFields.tsx

```diff
@@ -33,13 +33,13 @@ export class DetailsRowFields extends BaseComponent<IDetailsRowFieldsProps, IDet
             role={column.isRowHeader ? 'rowheader' : 'gridcell'}
             aria-colindex={columnIndex + columnStartIndex + 1}
             className={css(
-              rowClassNames.cell,
               column.className,
               column.isMultiline && rowClassNames.isMultiline,
               column.isRowHeader && rowClassNames.isRowHeader,
               column.isPadded && rowClassNames.isPadded,
               column.isIconOnly && shimmer && rowClassNames.shimmerIconPlaceholder,
-              shimmer && rowClassNames.shimmer
+              shimmer && rowClassNames.shimmer,
+              rowClassNames.cell
```

Comment:
I think this ties to @phkuo comment.. will this get worked out such that order is not important once everything is converted? Styling ordering here shouldn't matter, so I'm wondering if shimmer and cell should be mutually exclusive. (I did some work with links in Pivot with exclusion [here](https://github.com/OfficeDev/office-ui-fabric-react/pull/5324/files#diff-4714de560076bfda4e0b0272d915f1adR121)